openshift-container-platform-4/policies/IA-Identification_and_Authentication/component.yaml

```diff
@@ -192,27 +190,24 @@
 - control_key: IA-3 (3)
   standard_key: NIST-800-53
   covered_by: []
-  implementation_status: planned
+  implementation_status: partial
   narrative:
     - key: a
       text: |
-        A control response is planned. Engineering progress can be tracked via:
-
-        https://issues.redhat.com/browse/CMP-319
+        The system does not provide addresses to devices.  However, it does utilize a private IP space for 
+        communicable endpoints.  Within this space address lease parameters are irrelevant because the endpoints
+        themselves are ephemeral.  A single endpoint instance cannot obtain multiple addresses.
     - key: b
       text: |
-        A control response is planned. Engineering progress can be tracked via:
-
-        https://issues.redhat.com/browse/CMP-319
+        Addresses are tied to deployed communicable endpoints, which are audited.
```

Comment:
Unsure what this means. Could detail be given around how OpenShift containers are ephemeral, and when provisioned or redeployed, receive a new IP address/lease?